content/news/2024_07_27_GRASS_GIS_8_4_0_released.md

```diff
@@ -0,0 +1,89 @@
+  chicken and egg problems) - also for Jupyter notebooks.
+- GUI: New easy command history navigation through the History browser panel.
+- New [GRASS GIS Programming Style Guide](https://github.com/OSGeo/grass/blob/main/doc/development/style_guide.md).
+- For r.univar, the computation extended statistics is now parallelized.
+- JSON output format support (format="json") has been added in multiple tools
+  (e.g., r.report, r.info, v.db.select, t.rast.list, etc.).
+- A new support vector machine image classification (SVM) has been added in i.svm.
+
+Translations continue in
+[Weblate](https://weblate.osgeo.org/projects/grass-gis/), which
+automatically creates pull requests with the translated chunks.
+We'd like to **thank the translators of all languages** for their
+ongoing support!
+
+### Full list of changes and contributors
+
+For all 520 changes, see our detailed announcement with the full list of
+features and bugs fixed at
+[GitHub / Releases / 8.4.0](https://github.com/OSGeo/grass/releases/tag/8.4.0).
+
+Thanks to all contributors!
+
+<a href="/images/news/grass840_XXX.png">
+  <img src="/images/news/grass840_XXX..png"
+   alt="GRASS GIS 8.4.0"
+   title="GRASS GIS 8.4.0"
+   width="65%" style="float:right;padding-left:15px;padding-top:15px">
+</a>
+
+## Software downloads
+
+#### Binaries/Installers download
+
+- Windows
+  - [64bit standalone installer](https://grass.osgeo.org/grass84/binary/mswindows/native/WinGRASS-8.4.0-1-Setup.exe)
+- macOS
+  - [Official Application Bundle](https://cmbarton.github.io/grass-mac/download/)
+- Linux
+  - [Debian](https://tracker.debian.org/pkg/grass)
+  - [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable)
+  - [Fedora/Centos/EPEL](https://src.fedoraproject.org/rpms/grass)
+  - [Gentoo](https://packages.gentoo.org/packages/sci-geosciences/grass)
+
+Further binary packages for other platforms and distributions will follow shortly,
+please check at [software downloads](/download/software/).
+
+#### Source code download
+
+- <https://grass.osgeo.org/grass84/source/>
+- <https://grass.osgeo.org/grass84/source/grass-8.4.0.tar.gz>
+- [From GitHub](https://github.com/OSGeo/grass/releases/tag/8.4.0)
+- DOI: [10.5281/zenodo.13102854](https://doi.org/10.5281/zenodo.13102854)
+
+First time users may explore the [**first steps tutorial**](/learn/) after
+installation.
+
+**About GRASS GIS**
+
+The Geographic Resources Analysis Support System
+([https://grass.osgeo.org/](/)), commonly referred to as GRASS GIS, is
+an Open Source Geographic Information System providing powerful raster,
+vector and [geospatial processing capabilities](https://grass.osgeo.org/learn/overview/).
+It can be used either as a stand-alone application, as backend for other
+software packages such as QGIS and R, or in the cloud. It is
+distributed freely under the terms of the GNU General Public License (GPL).
+GRASS GIS is a founding member of the Open Source Geospatial Foundation (OSGeo).
```

Comment:
Can we remove this chunk, please? :pray:

Reply:
Why should it go? It is pretty standard for press releases

Reply:
Well, the news is on our website and the press release will point to it, right? So, I think a self description is not needed there. I don't see other projects having such a paragraph in their release notes either. 

What about replacing it with a *More about GRASS GIS* section and a bullet point list of links? Something like:

```
#### More about GRASS GIS

- See our list of [tutorials](https://grass.osgeo.org/learn/tutorials/).
- See GRASS [tools manual pages](https://grass.osgeo.org/grass-stable/manuals/index.html).
- [Get in touch with us](https://grass.osgeo.org/support/community/).
- Feel like supporting the project? [Donate](https://opencollective.com/osgeo/projects/grass/contribute).
```

Reply:
Traditionally, the press release was sent to multiple portals etc. As this no longer really happens (my lack of energy in this regard) I have reconsidered your suggestion and the new list of items.

Reply:
Thanks! I replaced it now

![image](https://github.com/user-attachments/assets/0a90e682-abcd-4e74-939b-4a7ede4b8bdf)

Reply:
Thanks a lot!